pkg/controller/federatedtypeconfig/controller.go

```diff
@@ -66,83 +64,214 @@ type FederatedTypeConfigController struct {
 	// Map of running sync controllers keyed by qualified target type
 	stopChannels map[string]chan struct{}
 	lock         sync.RWMutex
+
+	// For triggering reconciliation of a single resource. This is
+	// used when there is an add/update/delete operation on a resource
+	// in federated API server.
+	deliverer *util.DelayingDeliverer
+
+	// Store for the FederatedTypeConfig objects
+	store cache.Store
+	// Informer for the FederatedTypeConfig objects
+	controller cache.Controller
+
+	// Work queue allowing parallel processing of resources
+	workQueue workqueue.Interface
+
+	// Backoff manager
+	backoff *flowcontrol.Backoff
+
+	reviewDelay time.Duration
```

Comment:
`reviewDelay` and `smallDelay` are not being used.

Reply:
Done